boards/FSM/Src/App/states/App_AirClosedState.c

```diff
@@ -18,13 +18,49 @@ static void AirClosedStateRunOnTick(struct StateMachine *const state_machine)
         App_FsmWorld_GetPrimaryFlowMeter(world);
     struct FlowMeter *secondary_flow_meter =
         App_FsmWorld_GetSecondaryFlowMeter(world);
+    struct WheelSpeedSensor *left_wheel_speed_sensor =
+        App_FsmWorld_GetLeftWheelSpeedSensor(world);
+    struct WheelSpeedSensor *right_wheel_speed_sensor =
+        App_FsmWorld_GetRightWheelSpeedSensor(world);
 
     App_FlowMeter_Tick(primary_flow_meter);
     App_FlowMeter_Tick(secondary_flow_meter);
+
     App_SetPeriodicCanSignals_PrimaryFlowMeter(
         can_tx_interface, primary_flow_meter);
     App_SetPeriodicCanSignals_SecondaryFlowMeter(
         can_tx_interface, secondary_flow_meter);
+
+    const float left_wheel_speed =
+        App_Wheel_GetWheelSpeed(left_wheel_speed_sensor);
+    const float right_wheel_speed =
+        App_Wheel_GetWheelSpeed(right_wheel_speed_sensor);
+    App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED(
+        can_tx_interface, left_wheel_speed);
+    App_CanTx_SetPeriodicSignal_RIGHT_WHEEL_SPEED(
+        can_tx_interface, right_wheel_speed);
+
+    if (left_wheel_speed > App_Wheel_GetThreshold(left_wheel_speed_sensor))
+    {
+        App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED_OUT_OF_RANGE(
+            can_tx_interface, true);
+    }
+    else
+    {
+        App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED_OUT_OF_RANGE(
+            can_tx_interface, false);
+    }
+
+    if (right_wheel_speed > App_Wheel_GetThreshold(right_wheel_speed_sensor))
```

Comment:
```suggestion
    if (right_wheel_speed > App_Wheel_GetThreshold(right_wheel_speed_sensor))
```

Threshold is not clear because it can be lower threshold, upper threshold or both. I would rename it.